src/Model/List/ArrayList.php

```diff
@@ -455,6 +455,9 @@ protected function parseSortColumn($column, $direction = null)
      */
     public function sort(...$args): static
     {
+        if ($args = null) {
+            return $this;
+        }
```

Comment:
Setting sort to `null` means "don't perform sort operations".
Since `ArrayList` sorts at the time the method is called instead of at iteration time, passing in `null` should just do nothing.

Without this fix, we end up hitting `new InvalidArgumentException("Bad arguments passed to sort()")` which is:
1. inconsistent with other list implementations
2. problematic if a list which is usually a `DataList` has been swapped out with `ArrayList` in a place where typing isn't very strict

It was causing a test to fail because of 2 above.